benchmarks/bench_linalg_decompositions.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Worth a short docstring to say why KAK decomposition is the important thing to benchmark?

Reply:
Only KAK decomposition is currently benchmarked because it historically caused a problem of being too slow ([#1124](https://github.com/quantumlib/Cirq/issues/1124)) and this PR only ports the existing benchmarks to ASV. 

In future, we ideally should add benchmarks for other decompositions as well? I don't have a good reason as to why only KAK decomposition is important.

Reply:
Added an issue to track this: https://github.com/quantumlib/Cirq/issues/3840

Reply:
added a docstring.